commit 24fd0551333e7eded87b64dd36062da3df2f6380
Author: Explorer09 <explorer09@gmail.com>
Date:   Mon Sep 4 10:47:33 2017 +0800

    build: AC_USE_SYSTEM_EXTENSIONS in configure.ac.
    
    This would, e.g. define _GNU_SOURCE in config.h, enabling the
    reallocarray() prototype in glibc 2.26+ on Linux systems with that
    version of glibc.
    
    Fixes #241.

diff --git a/configure.ac b/configure.ac
index c6f12d6..3c977a4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -25,8 +25,10 @@
 # autoconf requirements and initialization
 
 AC_INIT([the fast lexical analyser generator],[2.6.4],[flex-help@lists.sourceforge.net],[flex])
+AC_PREREQ([2.60])
 AC_CONFIG_SRCDIR([src/scan.l])
 AC_CONFIG_AUX_DIR([build-aux])
+AC_USE_SYSTEM_EXTENSIONS
 LT_INIT
 AM_INIT_AUTOMAKE([1.15 -Wno-portability foreign std-options dist-lzip parallel-tests subdir-objects])
 AC_CONFIG_HEADER([src/config.h])
